proposals/stf-general-investment.md

```diff
@@ -68,7 +68,7 @@ Ever more individuals and organizations rely on it, but our capacity to protect
 
 ## Please provide a brief overview of projects that depend on your technology. 300 words
 
-- There are [40k GitHub repositories using the Nix language](https://github.com/search?q=language%3ANix&type=repositories&s=&o=desc)
+- There are [12k active GitHub repositories using the Nix language](https://github.com/search?q=language%3ANix+archived%3Afalse&type=repositories&s=&o=desc)
```

Comment:
I can't find `12k` when I click the link. How did you managed to get that number?

Reply:
That's what it shows to me... I don't know either.


![image](https://github.com/user-attachments/assets/b3035ab5-5e9e-4a66-b403-d742acad2e03)

Reply:
Fun, now I have something completely different... I had 2.1K the first time, now:

![image](https://github.com/user-attachments/assets/5d08389f-fcfa-4a22-acc3-9a11059601e0)

Reply:
When this line was first added, it showed me 40k!

Reply:
Mmmh I'm really wondering if those metrics are safe.

Reply:
For sure they aren't but it gives us an idea there are more than just a handful of projects on GitHub using Nix but more like thousands.